tests/handlers/test_metr_widgets.py

```diff
@@ -0,0 +1,33 @@
+from redash import models
+from tests import BaseTestCase
+
+class TestMetrWidgetTagsResource(BaseTestCase):
```

Comment:
Can you add tests for the permissions?

Reply:
thanks, please find tests added [here in this commit ](https://github.com/metr-systems/redash/pull/4/commits/9dbea40990cc4f323ce4be3162505e40fe7acc3b)